### ISSUE TITLE
'Always allow' for Autoplay Media is not automatically playing media

### ISSUE THREAD

**Issue by LaurenWags** [brave/browser-laptop#11069]
### Description
When navigating to http://beirutnights.com/ using a clean profile in 0.19.11, the Autoplay Media setting is 'Always allow' but the site does not automatically play. This did automatically play in 0.18.36.

### Steps to Reproduce

  1. Use clean profile/make sure your setting for Autoplay Media is 'always allow'
  2. Navigate to http://beirutnights.com/
  3. Media is not played automatically.

**Actual result:**
Media is not played automatically

**Expected result:**
Media should be played automatically

**Reproduces how often:**  100%

### Brave Version
Brave | 0.19.11
rev | 3252249
Muon | 4.4.19

**Reproducible on current live release:**
No, works in 0.18.36

### Additional Information

cc: @bsclifton

**Comment by darkdh** [comment]
FWIW, this bug is irrelevant to autoplay because `appActions.autoplayBlocked` doesn't get triggered. In other words, nothing blocked by autoplay.

**Comment by zentagonist** [comment]
Needs more investigation before can be resolved or addressed by the team.

**Comment by bsclifton** [comment]
Possibly related to https://github.com/brave/browser-laptop/issues/11042

**Comment by alexwykoff** [comment]
In 0.19.27, autoplay default value is set to 'always allow'.

When visiting https://www.vox.com/2017/10/1/16391722/snl-papyrus-avatar-ryan-gosling?utm_campaign=vox&utm_content=chorus&utm_medium=social&utm_source=twitter the video does not autoplay. 
Clicking on the embedded YouTube video to start playback adds a per-site setting of 'always allow' for vox.com. No dialog is presented to the user to inform them this per-site setting was made, or even asking permission to do such an action.

**Comment by darkdh** [comment]
Actually I take my previous comment back because I just saw the exception thrown from chromium `DOM Exception: play() can only be initiated by a user gesture`

**Comment by luixxiul** [comment]
Closing in favor of #11527.
